manager/integration/tests/test_basic.py

```diff
@@ -708,6 +708,99 @@ def test_backup_block_deletion(set_random_backupstore, client, core_api, volume_
     delete_backup_volume(client, volume_name)
 
 
+def test_backup_block_deletion_error_then_dr_restore(set_random_backupstore, client, core_api, volume_name):  # NOQA
+    """
+    Test backup block deletion
+
+    Context:
+
+    We want to make sure that when the block deletion for the last backup is
+    aborted by operations such as backups in progress, the DR volume will still
+    pick up the correct last backup.
+
+    Steps:
+
+    1.  Create a volume and attach to the current node.
+    2.  Write 4 MB to the beginning of the volume (2 x 2MB backup blocks).
+    3.  Create backup(0) of the volume.
+    4.  Overwrite backup(0) blocks of data on the volume.
+    5.  Create backup(1) of the volume.
+    6.  Verify backup block count == 3.
+    7.  Create DR volume from backup(1).
+    8.  Verify DR volume last backup is backup(1).
+    9.  Create an artificial in progress backup.cfg file.
+    10. Delete backup(1).
+    11. Verify backup block count == 3 (because of the in progress backup).
+    12. Delete the artificial in progress backup.cfg file.
+    13. Verify DR volume last backup is backup(0) or backup(1).
+        s3 -> backup(0), nfs -> backup(1).
```

Comment:
Why is there a difference?

Reply:
Actually, I am not entirely sure, will check the test case again.

Reply:
Cannot reproduce anymore, removed. Test result in https://github.com/longhorn/longhorn-tests/pull/592#issuecomment-819399339.